src/poliastro/tests/test_maneuver.py

```diff
@@ -55,7 +55,7 @@ def test_hohmann_maneuver():
 
     man = Maneuver.hohmann(ss_i, Earth.R + alt_f)
 
-    assert_quantity_allclose(ss_i.apply_maneuver(man).ecc, 0 * u.one, atol=1e-15 * u.one)
+    assert_quantity_allclose(ss_i.apply_maneuver(man).ecc, 0 * u.one, atol=1e-14 * u.one)
```

Comment:
Yes, the added precision for K parameter in DE430 made the test fail. Since the documentation of the test quotes a reference for the expected values, I thought that relaxing the tolerance was a better approach than changing the expected values.